src/mlpack/tests/main_tests/preprocess_split_test.cpp

```diff
@@ -357,3 +357,26 @@ TEST_CASE_METHOD(
   REQUIRE(static_cast<uvec>(find(
       params.Get<arma::Mat<size_t>>("test_labels") == 2)).n_rows == 20);
 }
+
+/**
+ * Check that if training size is not zero.
+ */
+TEST_CASE_METHOD(
+    PreprocessSplitTestFixture, "PreprocessSplitNonEmptyInputTest",
+    "[PreprocessSplitMainTest][BindingTests]")
+{
+  // Load custom dataset.
+  arma::mat inputData;
+  arma::Mat<size_t> labels;
+  if (!data::Load("vc2.csv", inputData))
+    FAIL("Cannot load train dataset vc2.csv!");
+  if (!data::Load("vc2_labels.txt", labels))
+    FAIL("Unable to load label dataset vc2_labels.txt!");
+
+  RUN_BINDING();
+
+  // Now check that the input data is not empty.
+  REQUIRE(inputData.is_empty() == false);
```

Comment:
What is this meant to be checking?  Shouldn't you be checking that an exception is thrown?

Reply:
yes, my bad!